setup.py

```diff
@@ -1,14 +1,27 @@
+import os
+
 from setuptools import setup, find_packages
 
+BASE_PATH = os.path.dirname(__file__)
+
 # https://packaging.python.org/guides/single-sourcing-package-version/
 version = {}
 with open("./colin/version.py") as fp:
     exec(fp.read(), version)
 
+
+def get_long_description():
```

Comment:
This one is easier. https://github.com/devassistant/devassistant/blob/master/setup.py#L107